core/eval.go

```diff
@@ -2052,6 +2052,19 @@ func evalEXISTS(args []string, store *Store) []byte {
 }
 
 func executeCommand(cmd *RedisCmd, c *Client, store *Store) []byte {
+	// For running redis tests
+	if cmd.Cmd == "SELECT" || cmd.Cmd == "FUNCTION" ||
```

Comment:
We can introduce flag of some sort to make server enter in `test` mode so that we can skip these commands for that case or alter some other config if required.

Reply:
Yes, we will move it to a config files where we will have a map of just pass go commands. Thanks for the suggestion